### ISSUE TITLE
Won't open

### ISSUE THREAD

**Issue by ghost** [rice-cracker-2234/BetterSynapse#7]
Just says something about how SLInjector isn't supported for Windows 10 when I run the UI, but when I run the actuall Synapse it works perfectly

**Comment by KryptoDemon** [comment]
it was a synapse error, dont worry about it
it's fixed now

**Comment by Pookiebeans** [comment]
Yeah this Slinjector crap is making it not work and my anti virus is off help!!!
